exporter/splunkhecexporter/client_test.go

```diff
@@ -48,12 +48,11 @@ func createMetricsData(numberOfDataPoints int) pdata.Metrics {
 	keys := []string{"k0", "k1"}
 	values := []string{"v0", "v1"}
 
-	unixSecs := int64(1574092046)
-	unixNSecs := int64(11 * time.Millisecond)
 	doubleVal := 1234.5678
 	var metrics []*metricspb.Metric
 	for i := 0; i < numberOfDataPoints; i++ {
-		tsUnix := time.Unix(unixSecs+int64(i), unixNSecs)
+		tsUnix := time.Unix(int64(i), int64(i)*time.Millisecond.Nanoseconds())
```

Comment:
I adjusted these tests so that we could test logs/metrics for 0-value timestamp event behaviour. Is this okay?